packages/trader/src/Modules/CFD/Components/cfd-financial-stp-pending-dialog.jsx

```diff
@@ -3,9 +3,15 @@ import { Modal, Button } from '@deriv/components';
 import { Localize, localize } from '@deriv/translations';
 import { connect } from 'Stores/connect';
 
-const CFDFinancialStpPendingDialog = ({ enableApp, disableApp, toggleModal, is_cfd_pending_dialog_open }) => (
+const CFDFinancialStpPendingDialog = ({
+    enableApp,
+    disableApp,
+    toggleModal,
+    is_cfd_pending_dialog_open,
+    is_fully_authenticated,
+}) => (
     <Modal
-        title={localize('Thanks for submitting your documents!')}
+        title={is_fully_authenticated ? ' ' : localize('Thanks for submitting your documents!')}
```

Comment:
```suggestion
        title={is_fully_authenticated && localize('Thanks for submitting your documents!')}
```

Reply:
I added an empty space to the title prop, otherwise if I leave it completely empty, it will break the modal and everything will shift to the top. Adding the empty space will keep everything perfectly aligned. Can I keep it this way or should I find a different approach?

Reply:
I forgot to add, that there is supposed to be no title when the user is fully authenticated. Only a different main message should be displayed in the modal.